examples/basic_bitcoin/test/tests.ts

```diff
@@ -320,27 +323,30 @@ function getTotalOutput(tx: Transaction): number {
     }, 0);
 }
 
-async function getP2pkhAddress(origin: string): Promise<string> {
-    const response = await fetch(`${origin}/get-p2pkh-address`, {
+export async function getP2wpkhAddress(origin: string): Promise<string> {
```

Comment:
p2pkh addresses are now gone for some reason

Reply:
done